### DISCUSSION TITLE
[Feature Request] #103 Use Database for song information

### DISCUSSION THREAD

**Discussion by RecursiveFunctions** [vicwomg/pikaraoke#296] [Q&A] [answered]
Continuing off from #103 , one way to go about this would be to use existing metadata of a song to complete the rest of it. 

Not sure if the current yt-dlp used in pikaraoke does this, but there _is_ a fork of it that is maintained and does pull the metadata automatically.

**Comment by safepay** [comment, marked as answer]
PR #792 introduces a db.
iTunes metadata is a forthcoming feature in dev.
